src/components/SidebarNav/sidebarNav.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useRef, useEffect } from 'react';
+import NavMenu from '../../common/navMenu';
+import { SidebarBlock, HamburgerSpan, SideBarNavWrapper } from './sidebarNavComponents';
+import SearchBar from '../SearchBar/searchBar';
+
+const SidebarNav = () => {
+  const [isSidebar, initSidebar] = useState(false)
```

Comment:
The name convention for the function in the `useState` is `set...`, in this case, should be `[isSidebar, setIsSidebar]`.